config/muxcable.py

```diff
@@ -188,3 +189,32 @@ def mode(state, port, json_output):
                 click.echo(tabulate(data, headers=headers))
 
         sys.exit(CONFIG_SUCCESSFUL)
+
+
+@muxcable.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+@click.argument('mode_value', required=True, default=None, type=click.INT)
+@click.argument('lane_map', required=True, default=None, type=click.INT)
+def prbs(port, target, mode_value, lane_map):
+
+    res = y_cable.enable_prbs_mode(port, target, mode_value, lane_map)
+    if res != True:
+        click.echo("PRBS config unsuccesful")
+        sys.exit(CONFIG_FAIL)
+    click.echo("PRBS config sucessful")
+    sys.exit(0)
```

Comment:
Why return 0 here, vs. `CONFIG_SUCCESSFUL` as done in other locations? And why is `CONFIG_SUCCESSFUL` 100?

Reply:
changed it to `CONFIG_SUCCESSFUL`
the return codes were decided to distinguish hwproxy from normal error codes which are non zero. 
On that front I defined all the codes with numbers from 100.

Reply:
> the return codes were decided to distinguish hwproxy from normal error codes which are non zero.

But why a non-zero return code for success?

Reply:
initially I had used 0 for success results and non zero otherwise and that was in accordance to other cli commands.  After few rounds of discussion hwproxy insisted to have all codes start from 100, just as a convention.

Reply:
:/ OK, I guess, but we return 0 pretty much everywhere else on success, so this isn't consistent. It's fine for now, but we might want to revisit it later.

Reply:
Also, it looks like this one is still `sys.exit(0)`

Reply:
fixed